### PR TITLE
List of external packages and their license

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

**Comment by jsancio** [comment]
cc @spahl

File: EXTERNAL_LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# List of external packages and their license
```
Comment (BenWhitehead) [inline EXTERNAL_LICENSE.md]: This sort of file is commonly referred to as a `NOTICE` file. See Spark for an example: https://github.com/apache/spark/blob/master/NOTICE

Comment (jsancio) [inline EXTERNAL_LICENSE.md]: Done.

**Comment by spahl** [comment]
Looks good sorry for the delay.

**Pull request merged**